spec/models/manageiq/providers/cloud_manager/provision_workflow_spec.rb

```diff
@@ -74,7 +74,10 @@
       FactoryGirl.create(:classification_cost_center_with_tags)
       admin.current_group.entitlement = Entitlement.create!(:filters => {'managed'   => [['/managed/cc/001']],
                                                                          'belongsto' => []})
-
+      2.times do
+        kp = ManageIQ::Providers::CloudManager::AuthKeyPair.create(:name => "auth_1")
```

Comment:
I realize this won't matter but if we are going to name the elements with a number should we give them unique names?

Maybe:
```ruby
2.times do |i|
  kp = ManageIQ::Providers::CloudManager::AuthKeyPair.create(:name => "auth_#{i}")
end
```

You can even use `i + 1` here if you don't want a zero element name.  😄